Prace_domowe/Praca_domowa2/Grupa4/Pawel_Wojciechowski/pd2_Pawel_Wojciechowski.ipynb

```diff
@@ -0,0 +1,1026 @@
+{
```

Comment:
<p>Fajen, to na zdecydowany i duży plus</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/93/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>